### PR TITLE
Network Load Balancer module

### PR DESCRIPTION
**Pull request by ludoo**

### CODE REVIEW

File: modules/net-nlb/health-check.tf [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Health check resource.
+
+locals {
+  hc       = var.health_check_config
+  hc_grpc  = try(local.hc.grpc, null) != null
```
Comment (juliocc) [inline modules/net-nlb/health-check.tf:21]: non-nullable and remove trys?

File: modules/net-nlb/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+output "groups" {
+  description = "Optional unmanaged instance group resources."
+  value       = google_compute_instance_group.unmanaged
+}
+
+output "health_check" {
+  description = "Auto-created health-check resource."
+  value       = try(google_compute_health_check.default.0, null)
```
Comment (juliocc) [inline modules/net-nlb/outputs.tf:61]: IMO `one()` instead of `try()` would make more sense here

**Review by juliocc — APPROVED** [review]
two small comments but LGTM

File: modules/net-nlb/outputs.tf (hunk 2 above)
Comment (ludoo) [reply modules/net-nlb/outputs.tf:61]: it might not be defined but passed in

**Pull request merged**